seed/static/seed/partials/geocode_modal.html

```diff
@@ -85,10 +86,15 @@ <h4 class="modal-title" id="exportModalLabel" ng-show="geocode_state === 'result
             </div>
         </div>
     </div>
+    <div ng-show="geocode_state === 'fail'">
+        <div class="alert alert-danger"  ng-if="error_message">
+            <p>{$ error_message | translate $}</p>
+        </div>
+    </div>
 
 </div>
 <div class="modal-footer">
     <button type="button" class="btn btn-primary" ng-click="geocode_buildings()" ng-show="geocode_state === 'verify' && has_api_key && has_enough_geocoding_columns" translate>Geocode</button>
     <button type="button" class="btn btn-default" ng-show="geocode_state == 'verify'" ng-click="cancel()" translate>Cancel</button>
-    <button type="button" class="btn btn-default" ng-show="geocode_state === 'result'" ng-click="close()" translate>Close</button>
+    <button type="button" class="btn btn-default" ng-show="geocode_state === 'result'||'fail'" ng-click="close()" translate>Close</button>
```

Comment:
I see this when I first open the geocoding modal now. 

<img width="558" alt="Screen Shot 2020-09-18 at 1 44 38 PM" src="https://user-images.githubusercontent.com/30608004/93638783-2138f880-f9b5-11ea-959e-b1023d035680.png">

The string 'fail' is truthy, so it ends up being `ng-show=<boolean> || <truthy>`.

Reply:
@adrian this relates to your last comment. Giving the same assumption where a user alters his/her mapquest api key in between importing a file and manually geocoding inventories, the geocoding attempts seen here makes sense to me. If the user proceed without correcting his/her key, error pops up upon them hitting "Geocode" button, in this case, also means geocoding for the 2nd time but manually.

Reply:
I'm not sure I understand what you mean. Let's have a call about these comments